aea/configurations/schemas/aea-config_schema.json

```diff
@@ -103,47 +103,5 @@
     "description": {
       "type": "string"
     }
-  },
```

Comment:
we are not going to copy `definitions` for every schema file. It is not maintainable. 
We introduced this workaround in PR #289. However, that is not a long-term solution, so if the problem still occurs on Windows we have to find another solution.